src/main/java/com/adaptavist/tm4j/jenkins/utils/GsonUtils.java

```diff
@@ -0,0 +1,28 @@
+package com.adaptavist.tm4j.jenkins.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializer;
+
+public class GsonUtils {
```

Comment:
We should probably make such conversion in the REST API/Backend. 

I think that it's reasonable to accept `50.0` as an integer value, for example.

Reply:
@johnathafelix  how  is it going to differentiate between a Number or a Decimal Number Custom fields?

Reply:
I agree. We can do this conversion in the API.

Reply:
@johnathafelix  got it!  .. Sorry,   I thought that all the numbers to `50.0`